KesaApplication/LoginViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  LoginViewController.swift
+//  KesaApplication
+//
+//  Created by Jiwoo Lim on 2016. 6. 1..
+//  Copyright © 2016년 kesa. All rights reserved.
+//
+
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    //Function to display error message
+    func displayAlertMessage(userMessage: String){
+        let myAlert = UIAlertController(title: "Error", message: userMessage, preferredStyle: UIAlertControllerStyle.Alert);
+        
+        let okAction = UIAlertAction(title: "OK", style: UIAlertActionStyle.Default, handler:nil);
+        
+        myAlert.addAction(okAction);
+        
+        self.presentViewController(myAlert, animated: true, completion: nil);
+    }
+
+    //Two text fields
+    @IBOutlet weak var userEmailTextField: UITextField!
+    @IBOutlet weak var userPasswordTextField: UITextField!
+    
+    @IBAction func loginButtonTapped(sender: AnyObject) {
+        
+        let userEmail = userEmailTextField.text;
+        let userPassword = userPasswordTextField.text;
+        
+        let userEmailStored = NSUserDefaults.standardUserDefaults().stringForKey("userEmail");
+        let userPasswordStored = NSUserDefaults.standardUserDefaults().stringForKey("userPassword");
+        
+        //Hide Keyboard when login button clicked
+        self.userEmailTextField.resignFirstResponder()
+        
+        self.userPasswordTextField.resignFirstResponder()
+        
+        
+        //Login is successfull
+        if(userEmailStored == userEmail){
+            
+            if(userPasswordStored == userPassword){
+                
+                
+                NSUserDefaults.standardUserDefaults().synchronize();
+                self.dismissViewControllerAnimated(true, completion: nil);
+                
+            }
+            
+        }else{  //Login unsuccessful
+            
+            //display alert message
+            displayAlertMessage("The email or password is incorrect");
+            return;
+        }
+    }
```

Comment:
This entire login successful scenario and unsuccessful scenario can be included in ResultHandler object.
Also, what does ``` NSUserDefaults.standardUserDefaults().synchronize(); ``` line do?